test/test_api.cpp

```diff
@@ -12,12 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <gtest/gtest.h>
```

Comment:
ah, that worked

On Mon, Oct 26, 2015 at 5:06 PM, Dirk Thomas <notifications@github.com>
wrote:

> In test/test_api.cpp
> <https://github.com/ros2/rttest/pull/12#discussion_r43070775>:
>
> > @@ -12,12 +12,13 @@
> >  // See the License for the specific language governing permissions and
> >  // limitations under the License.
> >
> > +#include <gtest/gtest.h>
>
> Have you tried using quotes instead?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ros2/rttest/pull/12/files#r43070775>.
>